src/main/fc/runtime_config.c

```diff
@@ -23,19 +23,19 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
-uint8_t armingFlags = 0;
-uint8_t stateFlags = 0;
-uint16_t flightModeFlags = 0;
+armingFlag_e armingFlags = 0;
+stateFlags_e stateFlags = 0;
+flightModeFlags_e flightModeFlags = 0;
```

Comment:
@martinbudden : `packed` enum will track required type size automatically, but it may cause problems elsewhere.

Also gdb is smart enough to print enum content symbolically, even to bitmasks ... 

```
enum t {
	a=1,
	b=2,
	c=4
};

int main(void) {
	enum t x = a|b;
	return 0;
}

(gdb) p x
$1 = (a | b)